llvm/lib/Target/AArch64/AArch64SystemOperands.td

```diff
@@ -1942,12 +1942,10 @@ def : RWSysReg<"PM",                0b11, 0b000, 0b0100, 0b0011, 0b001>;
 
 // 2023 ISA Extension
 // AArch64 Floating-point Mode Register controls behaviors of the FP8
-// instructions (FEAT_FPMR)
```

Comment:
The comment `(FEAT_FPMR)` should stay, because it is still true even if we are not going to gate the registers by feature.

Reply:
Done